activestorage/README.md

```diff
@@ -136,6 +136,40 @@ Active Storage, with its included JavaScript library, supports uploading directl
 | `direct-upload:end` | `<input>` | `{id, file}` | A direct upload has ended. |
 | `direct-uploads:end` | `<form>` | None | All direct uploads have ended. |
 
+## Download and installation
+
+The latest version of Active Storage can be installed with RubyGems, or with npm.
+
+### Installation with RubyGems
+
+```
+$ gem install activestorage
+```
+
+### Installation with npm
+
+```
+$ npm install --save activestorage
+```
+
+Source code can be downloaded as part of the Rails project on GitHub
+
+* https://github.com/rails/rails/tree/master/activestorage
+
```

Comment:
This seems to suggest that you can install Active Storage itself (rather than its direct upload JS) via RubyGems or npm.

Reply:
Thanks for the reviewing. I've swapped the top and bottom paragraphs. If this is not good solution, would you mind if teach me how to resolve this?